### DISCUSSION TITLE
Working With google/protobuf/struct.proto and Kafka Table Engine

### DISCUSSION THREAD

**Discussion by jonny7** [ClickHouse/ClickHouse#68912] [Q&A]
I recently raised this crash report https://github.com/ClickHouse/ClickHouse/issues/62467 that was fixed. Am I right to assume that there isn't a way to directly work with a protobuf message that uses `google/protobuf/struct.proto` field and the Kafka table engine?

**Comment by sundeepChandhoke** [comment]
Any update on this.  I want to use google/protobuf/struct.proto so I can get dynamic data in to JSON columns.  Is there a way to deserialize the struct.proto field into the new JSONObject?

**Reply by sundeepChandhoke** [reply]
Also @jonny7 did you find a workaround that you can share?

**Reply by jonny7** [reply]
We moved ingestion away from Clickhouse directly and into connectors
